### ISSUE TITLE
Relative skin CSS URLs loaded multiple times in DOM

### ISSUE THREAD

**Issue by pixelchutes** [jwplayer/jwplayer#1055]
The documentation recommends against configuring the skin via URL (e.g. skin.url), as this can impact player setup performance. Makes sense.

In the event you _need_ to rely on this method, note that if there are _multiple_ players (2+) on the page using the same configuration setup, the associated CSS will also be loaded just as many times as there are players (even when using the _same_ skin.url between player instances)

### Example:
_(assume 3 players with same config...)_

CONFIG:
```javascript
skin: {
    name: 'custom',
    url: '/path/to/custom.css' // Relative URL
}
```
RESULT:
```html
<head>
    <link type="text/css" rel="stylesheet" href="/path/to/custom.css">
    <link type="text/css" rel="stylesheet" href="/path/to/custom.css">
    <link type="text/css" rel="stylesheet" href="/path/to/custom.css">
    ...
</head>
```

However, provided you do _not_ use a relative URL, and enter a fully qualified URL (containing http(s)://) under the same circumstances, the CSS is luckily only loaded once.

CONFIG:
```javascript
skin: {
    name: 'custom',
    url: 'https://domain.com/path/to/custom.css' // Fully qualified URL
}
```
RESULT:
```html
<head>
    <link type="text/css" rel="stylesheet" href="https://domain.com/path/to/custom.css">
...
</head>
```

Duplicating relative `skin.url` multiple times in the DOM definitely feels like a bug. I'm just not sure if it's a browser-specific issue, or the way JW7 coding handles relative CSS skin links.

For original posting, see: https://support.jwplayer.com/customer/portal/questions/15957458-skin-css-url-loaded-multiple-times-jw-7-

**Comment by robwalch** [comment]
Hi @pixelchutes,

Thanks for the bug report. I'm not sure when we'll be able to get to this one. Since the css is relative to the page, maybe just include the css in the page header?

**Comment by pixelchutes** [comment]
No worries @robwalch, there's no rush. Totally fine using fully qualified URL as a work-around. Wanted to document my findings here for your reference.

**Comment by johnBartos** [comment]
May be fixed in 7.5.0 with the new `simple-style-loader`. I'll check it out when I get the chance

**Comment by pixelchutes** [comment]
@johnBartos,

From what I can tell (testing in v7.5.2), it seems this particular issue was resolved!

While I'm not sure how much it matters, the custom skin file is loaded in `<head>` _before_ the global / default inline styles (e.g. `<style type="text/css" data-jwplayer-id="all-players">`)
